src/spikeinterface/widgets/motion.py

```diff
@@ -96,55 +94,241 @@ def plot_matplotlib(self, data_plot, **backend_kwargs):
             ax.set_ylabel("Depth [um]")
 
 
+class DriftRasterMapWidget(BaseWidget):
+    """
+    Plot the drift raster map from peaks or a SortingAnalyzer.
+    The drift raster map is a scatter plot of the estimated peak depth vs time and it is
+    useful to visualize the drift over the course of the recording.
+
+    Parameters
+    ----------
+    peaks : np.array | None, default: None
+        The peaks array, with dtype ("sample_index", "channel_index", "amplitude", "segment_index"),
+        as returned by the `detect_peaks` or `correct_motion` functions.
+    peak_locations : np.array | None, default: None
+        The peak locations, with dtype ("x", "y") or ("x", "y", "z"), as returned by the
+        `localize_peaks` or `correct_motion` functions.
+    sorting_analyzer : SortingAnalyzer | None, default: None
+        The sorting analyzer object. To use this function, the `SortingAnalyzer` must have the
+        "spike_locations" extension computed.
+    direction : "x" or "y", default: "y"
+        The direction to display. "y" is the depth direction.
+    segment_index : int, default: None
+        The segment index to display.
+    recording : RecordingExtractor | None, default: None
+        The recording extractor object (only used to get "real" times).
+    segment_index : int, default: 0
+        The segment index to display.
+    sampling_frequency : float, default: None
+        The sampling frequency (needed if recording is None).
+    depth_lim : tuple or None, default: None
+        The min and max depth to display, if None (min and max of the recording).
+    scatter_decimate : int, default: None
+        If > 1, the scatter points are decimated.
+    color_amplitude : bool, default: True
+        If True, the color of the scatter points is the amplitude of the peaks.
+    cmap : str, default: "inferno"
+        The colormap to use for the amplitude.
+    color : str, default: "Gray"
+        The color of the scatter points if color_amplitude is False.
```

Comment:
something for us to discuss. We are super inconsistent with

`color_amplitude` vs `'color_amplitude'` vs ``color_amplitude`` (with ticks, but they are being rendered so you can't see them in this third option....)

Reply:
Right. I like ticks because it's clear they refer to arguments and " for strings. We need another unification effort here

Reply:
I agree with you. I like ticks to emphasize arguments. I'll open an issue for the doc team :)